pkg/rpc/client/wsclient.go

```diff
@@ -158,3 +207,73 @@ func (c *WSClient) makeWsRequest(r *request.Raw) (*response.Raw, error) {
 		return resp, nil
 	}
 }
+
+func (c *WSClient) performSubscription(params request.RawParams) (string, error) {
```

Comment:
`subscribe`/`unsubscribe` ?

Reply:
For some reason I tend to avoid naming methods like `Subscribe`/`subscribe`, it might be confusing. And this one is named so mostly because of already existing `performRequest`.